.github/workflows/tics.yml

```diff
@@ -0,0 +1,59 @@
+name: Linux
+
+on:
+    schedule:
+    # 0:00 UTC every Sunday
+    - cron: 0 0 * * 0
```

Comment:
Can we phase this a little off the round hour? For example `11 4 * * 0`? Still the night from Saturday to Sunday, but 4 hours later and irregular minutes.

Reply:
Sure. Can I ask why though?

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

> The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.